app/ecosystem/fixture.ts

```diff
@@ -150,5 +150,217 @@ export const projects: ProjectCardProps[] = [
+    categories: ['social'],
+    twitter: 'https://x.com/onopenxyz',
+    website: 'https://onopen.xyz/'
+  },
+  {
+    image: '/img/ecosystem/tachyswap.jpg',
+    title: 'TachySwap',
+    description: 'AMM based decentralized exchange.',
+    categories: ['defi'],
+    twitter: 'https://x.com/tachyswap',
+    website: 'https://tachyswap.org/'
+  },
+  {
+    image: '/img/ecosystem/cobweb-labs.jpg',
+    title: 'CobWeb Labs',
+    description: 've(3,3) DEX.',
+    categories: ['defi'],
+    twitter: 'https://x.com/CobWebLabs',
+    website: 'https://cobweblabs.xyz'
+  },
+  {
+    image: '/img/ecosystem/Tenderly.png',
+    title: 'Tenderly',
+    description:
+      'Full-stack Web3 infrastructure – node infra, exploration tools & dev environments.',
+    categories: ['dev-tools'],
+    twitter: 'https://x.com/TenderlyApp',
+    website: 'https://tenderly.co/'
+  },
+  {
+    image: '/img/ecosystem/Bithotel.jpg',
+    title: 'Bithotel',
+    description:
+      'Revolutionary social pixelart game (free-to-play!) on Mobile and PC.',
+    categories: ['gaming'],
+    twitter: 'https://x.com/playbithotel',
+    website: 'https://bithotel.io/#/'
+  },
+  {
+    image: '/img/ecosystem/TaskOn.svg',
+    title: 'TaskOn',
+    description:
+      'A Web3 task collaboration platform ideal for marketing, operations and community growth.',
+    categories: ['social'],
+    twitter: 'https://x.com/taskonxyz',
+    website: 'https://taskon.xyz/'
+  },
+  {
+    image: '/img/ecosystem/Questflow.jpg',
+    title: 'Questflow',
+    description:
+      'Decentralized Autonomous AI Agent Network - orchestrate multiple AI agents to take action on autopilot & dispatch real-world incentives to agents.',
+    categories: ['social'],
+    twitter: 'https://x.com/questflow',
+    website: 'https://www.questflow.ai/'
+  },
+  {
+    image: '/img/ecosystem/TheGraph.jpg',
+    title: 'The Graph',
+    description:
+      'A web3 protocol for organizing and accessing blockchain data.',
+    categories: ['infra'],
+    twitter: 'https://x.com/graphprotocol',
+    website: 'https://thegraph.com'
+  },
+  {
+    image: '/img/ecosystem/Omnisea.jpg',
+    title: 'Omnisea',
+    description: 'Omnichain NFT Launchpad powered by LayerZero.',
+    categories: ['nfts'],
+    twitter: 'https://x.com/omnisea',
+    website: 'https://omnisea.org/'
+  },
+  {
+    image: '/img/ecosystem/groupfi.jpg',
+    title: 'GroupFi',
+    description:
+      'A Web3 messaging protocol designed by the Tanglepay Team for seamless, feeless, on-chain and cross-application group communication. ',
+    categories: ['dev-tools'],
+    twitter: 'https://x.com/groupfi_ai',
+    website: 'https://groupfi.ai/'
+  },
+  {
+    image: '/img/ecosystem/ChessPlay.jpg',
+    title: 'Chess Play',
+    description:
+      'Chess meets blockchain. Your destination for innovative gameplay and secure transactions. Reshaping the future of on chain chess!',
```

Comment:
```suggestion
      'Chess meets blockchain: your destination for innovative gameplay and secure transactions, reshaping the future of onchain chess!',
```

Just to make these complete sentences.